src/auth/FormElement.ts

```diff
@@ -95,6 +95,15 @@ export default class GmfAuthForm extends GmfBaseElement {
       [hidden] {
         display: none !important;
       }
+      .login-spinner {
+        float: left;
+        margin-right: 20px;
+      }
+
+      i.fa-spin {
+        fill: black;
+        width: 1.3rem;
+      }
```

Comment:
It is not possible to move that in a specific exported variable, like with the `...GmfBaseElement.styles` (line 93) ? Because this css was in the "global" css before shadow dom. And now, it will be probably needed in multiple components, and we don't want to write the same code everytime, everywhere. (But maybe put it in another variable, because we don't want it in each component, like for the `...GmfBaseElement.styles`)

Reply:
In the original file it was on the class `.login-spinner` then it should be related only to the login, and I think that better to don't have too many shared CSS, as I know it's hard to manage it...

Reply:
Oh ok, in this case it's fine. But in the case it would be shared between 3 components or more, the question would be interesting.